packages/data-explorer/src/lib/api.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
```

Comment:
Maybe would could use 'transformResponse' to handle errors, could also to the 'login' redirect'

Reply:
validateStatus lijkt hiervoor bedoeld